www/components/Nav/index.tsx

```diff
@@ -243,10 +243,16 @@ const Nav = (props: Props) => {
               </div>
               <div className="hidden lg:flex items-center sm:space-x-3">
                 <a href="https://github.com/supabase/supabase" target="_blank">
-                  <Button type="default" icon={<IconStar />}>
+                  <Button
+                    className="sbui-btn sbui-btn-text sbui-btn--tiny sbui-btn--text-align-center"
+                    icon={<IconStar />}
+                  >
                     Star us on GitHub
                   </Button>
                 </a>
+                <a href="https://app.supabase.io/" target="_blank">
```

Comment:
What is the rationale for `target="_blank"` here? 🔎

Reply:
No specific reasons for `target="_blank"`. We can remove it so the user should remain on the same page while signing in?